content/en/docs/04.0/_index.en.md

```diff
@@ -89,6 +89,15 @@ Use the command `kubectl get` with the `-w` parameter in order to get the reques
 kubectl get pods -w --namespace <namespace>
 ```
 
+{{% alert title="Note" color="primary" %}}
+Instead of using the `-w` parameter you can also use the `watch` command which should be available for most Linux distributions:
```

Comment:
Both are right but maybe it sounds more natural this way as `watch ` is usually pre-installed and already available.

```suggestion
Instead of using the `-w` parameter you can also use the `watch` command which should be available on most Linux distributions:
```